[IMP] CRM: lead mining 17 (#9315)
odoo/documentation: content/applications/sales/crm/acquire_leads/lead_mining.rst

==============

After the :guilabel:`Lead Mining` setting is activated, a new button called :guilabel:`Generate
Leads` is available to use in the :guilabel:`CRM` pipeline (:menuselection:`CRM App --> Sales --> My
Pipeline`).

Lead mining requests are also available through :menuselection:`CRM --> Configuration --> Lead
Mining Requests`, or through :menuselection:`CRM --> Leads --> Leads`, where the :guilabel:`Generate
Leads` button is also available.

.. image:: lead_mining/generate-leads-button.png
   :align: center
   :alt: The Generate Leads button to use the lead mining feature.

Click the :guilabel:`Generate Leads` button, and a pop-up window appears, offering a variety of
criteria by which to generate leads.

.. image:: lead_mining/generate-leads-popup.png
   :align: center
   :alt: The pop-up window with the selection criteria in order to generate leads in Odoo.

Choose to generate leads for :guilabel:`Companies` to get company information only, or choose
|CC| to get company information and individual employee contact information.

.. note::
   When targeting |CC|, additional options are available to filter contacts based on

>    :guilabel:`Role` or :guilabel:`Seniority`.

Author: I left it as just "Role" because you only specify the "preferred role" *after* you filter by role

Additional filtering options include the following:

- :guilabel:`Countries`: filter leads based on the country (or countries) they are located in.
- :guilabel:`States`: further filter leads based on the state in which they are located, if
  applicable.
- :guilabel:`Industries`: filter leads based on the specific industry they work in.
- :guilabel:`Filter on Size`: tick this checkbox to specify the number of employees at the company.
  This generates a field labeled :guilabel:`Size`. Fill in the blanks to create a range for the
  desired company size.
- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to.
- :guilabel:`Salesperson`: choose which member of the Sales Team the leads will be assigned to.
- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found.

.. note::
   Make sure to be aware of the latest EU regulations when receiving contact information. Get more
   information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`_.

View leads
----------

After leads are generated, they are assigned to the designated salesperson and team. To view

> additional information regarding the lead, select one from the list and click to open it.

Reviewer: ```suggestion
additional information regarding the lead, select one from the list, and click to open it.
```

In the *chatter* thread for the lead, additional information is provided. This can include the
number of employees, the technology used by the company, the timezone, and direct contact
information.

.. image:: lead_mining/generated-lead.png
   :align: center
   :alt: The chatter thread of a newly generated lead.

.. note::
   If :guilabel:`Leads` are **not** enabled for the database, then leads are generated as

>    *opportunities* and added to the pipeline for the designated salesperson.

Reviewer: ```suggestion
   *opportunities*, and added to the pipeline for the designated salesperson.
```

   To enable the :guilabel:`Leads` feature, navigate to :menuselection:`CRM app --> Configuration

>    --> Settings` and tick the :guilabel:`Leads` checkbox.

Reviewer: ```suggestion
   --> Settings`, and tick the :guilabel:`Leads` checkbox. Then, click :guilabel:`Save`.
```

Pricing
=======

Lead mining is an *In-App Purchase* feature, and each generated lead costs one :ref:`credit
<in_app_purchase/credits>`.

.. important::
   Generating |CC| costs one additional credit for each contact generated. See here for complete
   pricing information: `Lead Generation by Odoo IAP
   <https://iap.odoo.com/iap/in-app-services/167?>`_.

> To buy credits, navigate to :menuselection:`CRM --> Configuration --> Settings`. In the

Reviewer: ```suggestion
To buy credits, navigate to :menuselection:`CRM app --> Configuration --> Settings`. In the
```

:guilabel:`Lead Generation` section, under the :guilabel:`Lead Mining` feature, click on
:guilabel:`Buy Credits`.

> Credits may also be purchased by navigating to the :menuselection:`Settings --> General Settings`.

Reviewer: ```suggestion
Credits may also be purchased by navigating to the :menuselection:`Settings app --> General Settings`.
```
Since the first thing seen when opening the Settings app *is* the General Settings page, is this additional step in the menuselection necessary? What do you think?

Author: Probably not, I will update :+1:

> In the :guilabel:`In-App Purchases` section, under the :guilabel:`Odoo IAP` feature, click on

Reviewer: ```suggestion
In the :guilabel:`Contacts` section, under the :guilabel:`Odoo IAP` feature, click on
```

==============

> After the :guilabel:`Lead Mining` setting is activated, a new button called :guilabel:`Generate

Reviewer 2: Nitpick– guilabels should only be used when the text is on the UI 
```suggestion
After the *Lead Mining* setting is activated, a new button called *Generate Leads*
```

> Leads` is available to use in the upper-left corner of the *CRM* :guilabel:`Pipeline`

Reviewer 2: ```suggestion
Leads` is available to use in the upper-left corner of the *CRM* *Pipeline*
```

Choose to generate leads for :guilabel:`Companies` to get company information only, or choose
|CC| to get company information and individual employee contact information.

.. note::
   When targeting |CC|, additional options are available to filter contacts based on
   :guilabel:`Role` or :guilabel:`Seniority`.

Additional filtering options include the following:

- :guilabel:`Countries`: filter leads based on the country (or countries) they are located in.
- :guilabel:`States`: further filter leads based on the state in which they are located, if
  applicable.
- :guilabel:`Industries`: filter leads based on the specific industry they work in.
- :guilabel:`Filter on Size`: tick this checkbox to specify the number of employees at the company.
  This generates a field labeled :guilabel:`Size`. Fill in the blanks to create a range for the
  desired company size.
- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to.
- :guilabel:`Salesperson`: choose which member of the Sales Team the leads will be assigned to.
- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found.

> .. note::

Reviewer 3: Given how important this is for the EU (and Odoo as whole), I'd change the block accordingly:
```suggestion
.. important::
```

   Make sure to be aware of the latest EU regulations when receiving contact information. Get more
   information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`_.

View leads
----------

After leads are generated, they are assigned to the designated salesperson and team. To view
additional information regarding the lead, select one from the list, and click to open it.

> In the *chatter* thread for the lead, additional information is provided. This can include the
> number of employees, the technology used by the company, the timezone, and direct contact
> information.

Reviewer 2: No need for italics formatting on _chatter_ IMO
```suggestion
In the chatter thread for the lead, additional information is provided. This can include the number
of employees, the technology used by the company, the timezone, and direct contact information.
```
